settings.gradle

```diff
@@ -1,7 +1,7 @@
 rootProject.name = 'nva-doi-registrar-client'
 include 'assign-doi-datacite'
-include 'dynamo-event-to-datacite-request'
 include 'datacite-doi-commons'
 include 'datacite-draft-doi-handler'
+include 'assign-doi-datacite-example'
```

Comment:
Assuming this goes away after merging with latest `develop`?

Reply:
My mistake when merging develop last time. Should be fixed now.